src/components/Feed/WrapToken.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
+import styled from 'styled-components'
+import {
+  Box,
+  Button,
+  GU,
+  Help,
+  LoadingRing,
+  useLayout,
+  textStyle,
+  useTheme,
+} from '@1hive/1hive-ui'
+
+import { useAppState } from '../../providers/AppState'
+
+import { formatTokenAmount } from '../../utils/token-utils'
+
+import wrappedIcon from '../../assets/wrappedIcon.svg'
+import unwrappedIcon from '../../assets/unwrappedIcon.svg'
+
+function WrapToken({ onUnwrapToken, onWrapToken }) {
+  const { layoutName } = useLayout()
+  const {
+    accountBalance: gardenTokenBalance,
+    stakeToken,
+    wrappableAccountBalance,
+    wrappableToken,
+  } = useAppState()
+
+  const theme = useTheme()
+  const compactMode = layoutName === 'small' || layoutName === 'medium'
+
+  return (
+    <Box
+      css={`
+        ${!compactMode && `margin-bottom: ${3 * GU}px;`}
+      `}
+    >
+      {gardenTokenBalance.eq(-1) || wrappableAccountBalance.eq(-1) ? (
+        <div
+          css={`
+            width: 100%;
+            height: ${21 * GU}px;
```

Comment:
After loading i see that the height is reajusted 
```suggestion
            height: ${22 * GU}px;
```